src/main/java/seedu/address/model/entity/Team.java

```diff
@@ -325,6 +325,6 @@ public boolean isSameTeam(Team otherTeam) {
         }
 
         return this.name.equals(otherTeam.getName())
-                || this.id.equals(otherTeam.getId());
+                || this.projectName.equals(otherTeam.getProjectName());
```

Comment:
I think this should be `&&`

Reply:
I was thinking that both should also be unique, else it is possible for a team to have two different projects or for two different teams to have the same project name

Reply:
what do you think?

Reply:
So it should be && right?

Reply:
Oh but if is && then both must be the same to be equal, but either or should be unique?